config/co2eq_parameters.js

```diff
@@ -1,18 +1,51 @@
 var exports = module.exports = {};
 
-var defaultCo2eqFootprint = {
-    'biomass': 230,
-    'coal': 820,
-    'gas': 490,
-    'hydro': 24,
-    'nuclear': 12,
-    'oil': 650,
-    'solar': 45,
-    'wind': 12,
-    'geothermal': 24,
-    'unknown': 700, // assume conventional
-    'other': 700 // same as 'unknown'. Here for backward compatibility
-}; // in gCo2eq/kWh
+var defaultCo2eqFootprint = { // in gCo2eq/kWh
+    'biomass': {
+      'co2eq': 230,
```

Comment:
Country specific emissions (https://github.com/tmrowco/electricitymap/blob/fc15c466b9406513647a7aa219300ae5783b37b1/config/co2eq_parameters.js#L51) use the `value` key instead of `co2eq`. For sake of consistency, I think we should use the same.
Feel free to either pick `co2eq` or `value` (but let's keep both in sync).